modules/ROOT/pages/java-support.adoc

```diff
@@ -94,6 +96,19 @@ To upgrade Anypoint Connectors:
 . In Anypoint Studio, update the connectors in your app to the latest Java 17-compatible version. 
 . xref:mule-runtime::deploying.adoc[Deploy your applications] to a target running on Mule 4.6 or later.
 
+[[anypoint-studio]]
+== Anypoint Studio
+
+Starting with Anypoint Studio (Studio) 7.17, if your Mule app is running on Mule runtime 4.6 or later, you can change the target JVM at the Studio project level to upgrade or downgrade the JVM. When you compile your app, Studio 7.17 offers real-time guidance and alerts you if there are any incompatible connector versions or project mismatches to prevent deployment failures.
```

Comment:
These 4 sentences that start with "When" seem like bullet points, which might be more scannable?